### PR TITLE
System Bridge Integration

### PR DESCRIPTION
**Pull request by timmo001**
## Proposed change

Adds documentation for initial System Bridge integration.

## Type of change

- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/48156
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2351

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/system_bridge.markdown [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  bridge: device
+  command: python
+  arguments: '-V'
+```
+
+### Open Path/URL
+
+Open a URL or file on the server using the default application.
+
+[![Open your Home Assistant instance and show your service developer tools with a specific service selected.](https://my.home-assistant.io/badges/developer_call_service.svg)](https://my.home-assistant.io/redirect/developer_call_service/?service=system_bridge.open)
```
Comment (frenck) [inline source/_integrations/system_bridge.markdown]:
```suggestion
{% my developer_call_service service="system_bridge.open" title="Open your Home Assistant instance and show your service developer tools with a specific service selected." %}
```

File: source/_integrations/system_bridge.markdown [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+#### Examples
+
+```yaml
+service: system_bridge.open
+data:
+  bridge: device
+  path: 'C:\image.jpg'
```
Comment (frenck) [inline source/_integrations/system_bridge.markdown]:
```suggestion
service: system_bridge.open
data:
  bridge: "device"
  path: "C:\image.jpg"
```

File: source/_integrations/system_bridge.markdown [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+```
+
+```yaml
+service: system_bridge.open
+data:
+  bridge: device
+  path: 'https://home-assistant.io'
```
Comment (frenck) [inline source/_integrations/system_bridge.markdown]:
```suggestion
service: system_bridge.open
data:
  bridge: "device"
  path: "https://home-assistant.io"
```

File: source/_integrations/system_bridge.markdown [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+```
+
+```yaml
+service: system_bridge.open
+data:
+  bridge: device
+  path: 'steam://rungameid/814380'
```
Comment (frenck) [inline source/_integrations/system_bridge.markdown]:
```suggestion
service: system_bridge.open
data:
  bridge: "device"
  path: "steam://rungameid/814380"
```

File: source/_integrations/system_bridge.markdown [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+| Operating System | Version information of the Operating System         |
+| Load             | System load percentage                              |
+
+## Services
+
+### Send Command
+
+Sends a command to the server to run.
+
+[![Open your Home Assistant instance and show your service developer tools with a specific service selected.](https://my.home-assistant.io/badges/developer_call_service.svg)](https://my.home-assistant.io/redirect/developer_call_service/?service=system_bridge.send_command)
```
Comment (frenck) [inline source/_integrations/system_bridge.markdown]:
```suggestion
{% my developer_call_service service="system_bridge.send_command" title="Open your Home Assistant instance and show your service developer tools with a specific service selected." %}
```

File: source/_integrations/system_bridge.markdown [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+| Battery          | Battery level of the device                         |
+| CPU Speed        | The current CPU speed                               |
+| CPU Temperature  | The current temperature of the device               |
+| Filesystem(s)    | Space used for each drive letter / filesystem mount |
+| Operating System | Version information of the Operating System         |
+| Load             | System load percentage                              |
+
+## Services
+
+### Send Command
```
Comment (frenck) [inline source/_integrations/system_bridge.markdown]:
```suggestion
### Service `system_bridge.send_command`
```

File: source/_integrations/system_bridge.markdown [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+```yaml
+service: system_bridge.send_command
+data:
+  bridge: device
+  command: python
+  arguments: '-V'
+```
+
+### Open Path/URL
```
Comment (frenck) [inline source/_integrations/system_bridge.markdown]:
```suggestion
### Service `system_bridge.open`
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/system_bridge.markdown [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+  - System Monitor
+ha_release: 2021.4
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@timmo001'
+ha_domain: system_bridge
+ha_quality_scale: silver
+ha_platforms:
+  - sensor
```
Comment (MartinHjelmare) [inline source/_integrations/system_bridge.markdown:16]:
```suggestion
  - binary_sensor
  - sensor
```

File: source/_integrations/system_bridge.markdown [hunk 9]
```diff
@@ -0,0 +1,92 @@
+---
+title: System Bridge
+description: How to integrate the System Bridge integration into Home Assistant.
+ha_category:
+  - Sensor
+  - System Monitor
+ha_release: 2021.4
```
Comment (MartinHjelmare) [inline source/_integrations/system_bridge.markdown]:
```suggestion
ha_release: 2021.6
```

**Review by frenck — APPROVED** [review]
Thanks, @timmo001 👍

**Pull request merged**